src/main/cpp/DragonVision/DragonLimelight.cpp

```diff
@@ -70,15 +70,16 @@ DragonLimelight::DragonLimelight(
 
 /// @brief Assume that the current pipeline is AprilTag and that a target is detected
 /// @return -1 if the network table cannot be found
-int DragonLimelight::GetAprilTagID()
+std::optional<int> DragonLimelight::GetAprilTagID()
```

Comment:
I didn't think we were using limelights for april tag this year.

Reply:
We're not, but we had the functionality and this needed to be consistent with PhotonCamera